shuup/testing/theme/__init__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.xtheme import Theme
+
+
+class ShuupTestingTheme(Theme):
+    identifier = "shuup_testing"
+    name = _("Shuup Testing Theme")
+    author = "Shuup"
+    template_dir = "testing/"
+
+    plugins = [__name__ + ".plugins.TestHighlightPlugin"]
+
+    def get_view(self, view_name):
```

Comment:
It has. And IMO theme shouldn't have front theme views enabled by default. Also the import should be `import shuup.front.themes.views as views` to make that actually work.